pitop/processing/image_processor.py

```diff
@@ -0,0 +1,20 @@
+from pitop.processing.algorithms import find_line
+
+
+class ImageProcessor:
```

Comment:
Let's figure out how to handle image transforms as well as image processing, blah blah